californium-core/src/test/java/org/eclipse/californium/TestTools.java

```diff
@@ -128,4 +130,31 @@ public static boolean waitForCondition(long timeout, long interval, TimeUnit uni
 		}
 		return check.isFulFilled();
 	}
+	
+	/**
+	 * Find a free IP port
+	 */
+	public static int findFreePort() {
```

Comment:
Can you please explain, why that kind of "findFreePort" should be introduced again?
In my experience, this was one of the causes for "rare false reds".

Reply:
I just want a free port to be "sure" my hanshake will timeout.

Reply:
I will to that. This is probably even better as this ensure that nothing will answer on this port for the duration of the test.